### PR TITLE
AA-285 L05 - fix comments.

### PR DESCRIPTION
**Pull request by drortirosh**

### CODE REVIEW

File: contracts/core/BasePaymaster.sol [hunk 1]
```diff
@@ -72,12 +72,12 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
      *      it must also implement this method.
      * @param mode          - Enum with the following options:
      *                        opSucceeded - User operation succeeded.
-     *                        opReverted  - User op reverted. still has to pay for gas.
+     *                        opReverted  - User op reverted. The paymaster still has to pay for gas.
+     *                        (postOpReverted never used to call postOp)
```
Comment (forshtat) [inline contracts/core/BasePaymaster.sol]: I don't understand this line (`(postOpReverted never used to call postOp)`)

File: contracts/interfaces/IPaymaster.sol [hunk 2]
```diff
@@ -30,7 +30,7 @@ interface IPaymaster {
      * @return validationData - Signature and time-range of this operation, encoded the same as the return
      *                          value of validateUserOperation.
      *                          <20-byte> sigAuthorizer - 0 for valid signature, 1 to mark signature failure,
-     *                                                    otherwise, an address of an "authorizer" contract.
+     *                                                    other values are invalid for paymaster.
```
Comment (forshtat) [inline contracts/interfaces/IPaymaster.sol:33]: It is probably better to remove any mention of the `<20-byte> sigAuthorizer` as well, as it is not relevant for the Paymaster.

**Comment by forshtat** [comment]
What about these:
`The simulation functions both claim to always revert, but that is no longer accurate.`
`The paymasterAndData parameter is incorrectly described as a paymaster address followed by a token address.`

File: contracts/core/BaseAccount.sol [hunk 3]
```diff
@@ -69,7 +69,7 @@ abstract contract BaseAccount is IAccount {
      *                          (also hashes the entrypoint and chain id)
      * @return validationData - Signature and time-range of this operation.
      *                          <20-byte> sigAuthorizer - 0 for valid signature, 1 to mark signature failure,
-     *                              otherwise, an address of an "authorizer" contract.
+     *                                                    other values are invalid for paymaster.
```
Comment (shahafn) [inline contracts/core/BaseAccount.sol]: But this is BaseAccount..

File: contracts/interfaces/IAggregator.sol [hunk 4]
```diff
@@ -20,7 +20,8 @@ interface IAggregator {
 
     /**
      * Validate signature of a single userOp.
-     * This method is should be called by bundler after EntryPoint.simulateValidation() returns (reverts) with ValidationResultWithAggregation
+     * This method is should be called by bundler after EntryPointSimulation.simulateValidation() returns
```
Comment (shahafn) [inline contracts/interfaces/IAggregator.sol]:
```suggestion
     * This method should be called by bundler after EntryPointSimulation.simulateValidation() returns
```

File: contracts/interfaces/IPaymaster.sol [hunk 5]
```diff
@@ -46,7 +46,8 @@ interface IPaymaster {
      * Must verify sender is the entryPoint.
      * @param mode          - Enum with the following options:
      *                        opSucceeded - User operation succeeded.
-     *                        opReverted  - User op reverted. still has to pay for gas.
+     *                        opReverted  - User op reverted. The paymaster still has to pay for gas.
+     *                        (postOpReverted never used to call postOp)
```
Comment (shahafn) [inline contracts/interfaces/IPaymaster.sol]: I don't understand this sentence: `postOpReverted never used to call postOp`

File: contracts/interfaces/IStakeManager.sol [hunk 6]
```diff
@@ -37,8 +37,8 @@ interface IStakeManager {
      * @param stake           - Actual amount of ether staked for this entity.
      * @param unstakeDelaySec - Minimum delay to withdraw the stake.
      * @param withdrawTime    - First block timestamp where 'withdrawStake' will be callable, or zero if already locked.
-     * @dev Sizes were chosen so that (deposit,staked, stake) fit into one cell (used during handleOps)
-     *      and the rest fit into a 2nd cell.
+     * @dev Sizes were chosen so that deposit fit one cell (used during handleOp)
```
Comment (shahafn) [inline contracts/interfaces/IStakeManager.sol]:
```suggestion
     * @dev Sizes were chosen so that deposit fits into one cell (used during handleOp)
```

**Review by shahafn — DISMISSED** [review]

**Review by forshtat — APPROVED** [review]

**Pull request merged**